README.rst

```diff
@@ -71,7 +71,7 @@ Copyright Jonathan Hartley 2013. BSD 3-Clause license; see LICENSE file.
 Dependencies
 ============
 
-None, other than Python. Tested on Python 2.7, 3.4, 3.5 and 3.6.
+None, other than Python. Tested on Python 2.7 and 3.5+.
```

Comment:
I think I prefer to keep the explicit list of tested versions. When future versions get released, we can't say that they are tested until they are.

Reply:
Updated in https://github.com/tartley/colorama/pull/228/commits/9f03cb6c237ac420e60858f20ee775224cb5e186.